rocAL/rocAL/include/node_image_loader.h

```diff
@@ -44,7 +44,7 @@ class ImageLoaderNode: public Node
     /// \param load_batch_count Defines the quantum count of the images to be loaded. It's usually equal to the user's batch size.
     /// The loader will repeat images if necessary to be able to have images in multiples of the load_batch_count,
     /// for example if there are 10 images in the dataset and load_batch_count is 3, the loader repeats 2 images as if there are 12 images available.
-    void init(unsigned internal_shard_count, const std::string &source_path,const std::string &json_path, const std::map<std::string, std::string> feature_key_map, StorageType storage_type,
+    void init(unsigned internal_shard_count, const std::string &source_path,const std::string &json_path, unsigned sequence_length, unsigned step, unsigned stride, const std::map<std::string, std::string> feature_key_map, StorageType storage_type,
```

Comment:
Not sure why we need to pass video related parameters here? Please keep video and image readers seperate and remove unnecessary parameters from the API

Reply:
[Sequence Reader](https://github.com/fiona-gladwin/MIVisionX/blob/695f3083ddc8f7575853be502786f971a1b23803/rocAL/rocAL/source/rali_api_data_loaders.cpp#L328) uses the ImageLoaderNode, and it requires the parameters such as sequence_length, frame_step and frame_stride similar to Video Reader. Hence these 3 parameters has been set to 0 in all the image loader nodes (except for sequence reader).